File: Universal Windows Platform Solution/Phantom of the West/Phantom of the West/Visual Novel/Chapters/Chapter 11/ILeaveGuest.cs
using Phantom_of_the_West.Visual_Novel.Serialization.Event_Flagging;

namespace Phantom_of_the_West.Visual_Novel.Chapters.Chapter_11
{
	internal class ILeaveGuest : StoryChoice
	{
		internal ILeaveGuest() : base(StoryChoice.GetLocalizedString("ch11ILeaveGuest"), 11114)
		{
		}

		public override void Select()
		{
			PotWVN vn = PotWVN.MainVN;
			vn.EventFlags.Ch11HarassedAfterWorkout = WorkoutHarassment.Left;
			base.Select();
		}
	}
}

File: Android Project/app/src/main/java/com/christopherraleigh/phantomofthewest/visual_novel/chapters/chapter_11/ILeaveGuest.java
package com.christopherraleigh.phantomofthewest.visual_novel.chapters.chapter_11;

import com.christopherraleigh.phantomofthewest.R;
import com.christopherraleigh.phantomofthewest.visual_novel.PotWVN;
import com.christopherraleigh.phantomofthewest.visual_novel.chapters.StoryChoice;
import com.christopherraleigh.phantomofthewest.visual_novel.serialization.event_flagging.WorkoutHarassment;

/**
 * Created by christopherraleigh on 2016-05-22.
 */
public class ILeaveGuest extends StoryChoice {
    public ILeaveGuest() {
        super(R.string.ch11ILeaveGuest, 11114);
    }

    @Override
    public void select() {
        PotWVN vn = PotWVN.getMainVN();
        vn.getEventFlags().setCh11HarassedAfterWorkout(WorkoutHarassment.Left);
        super.select();
    }
}
